testsuite/features/build_validation/reposync/srv_sync_all_products.feature

```diff
@@ -66,7 +66,7 @@ Feature: Synchronize products in the products page of the Setup Wizard
   Scenario: Add SUSE Linux Enterprise Server 15 SP3
     When I follow the left menu "Admin > Setup Wizard > Products"
     And I wait until I do not see "Loading" text
-    And I enter "SUSE Linux Enterprise Server 15 SP3 x86_64" as the filtered product description
+    And I enter "SUSE Linux Enterprise Server 15 SP3" as the filtered product description
```

Comment:
why did we need to remove the architectures from all the other products?

Reply:
Just for consistency sake, and also because it's a quite "natural" workflow:

you select not specifying the arch, you get a list, you select one of them by specifying the arch

Reply:
ah I got it, it's just for searching, the selection is always by arch. Makes sense